internal/version/version.go

```diff
@@ -2,7 +2,7 @@ package version
 
 var (
 	// Version is the current version of the oras.
-	Version = "0.11.1"
+	Version = "0.11.0-experimental"
```

Comment:
@SteveLasker should we rename this to alpha similar to distribution?

Reply:
Should it be `0.11.1-experimental`?